crates/typed-store/src/rocks/mod.rs

```diff
@@ -1838,13 +1838,32 @@ pub fn default_db_options() -> DBOptions {
     }
 }
 
+/// Creates a default RocksDB option, to be used when RocksDB option is not specified..
+pub fn default_db_options() -> DBOptions {
+    let mut opt = base_db_options();
+
+    // Set options mostly similar to those used in optimize_for_point_lookup(),
+    // except non-default binary and hash index, to hopefully reduce lookup latencies
+    // without causing any regression for scanning, with slightly more memory usages.
+    // https://github.com/facebook/rocksdb/blob/11cb6af6e5009c51794641905ca40ce5beec7fee/options/options.cc#L611-L621
+    let mut block_options = BlockBasedOptions::default();
+    // Configure a 64MiB block cache.
+    block_options.set_block_cache(&Cache::new_lru_cache(64 << 20).unwrap());
+    // Set a bloomfilter with 1% false positive rate.
+    block_options.set_bloom_filter(10.0, false);
+    opt.options.set_block_based_table_factory(&block_options);
+    // Set memtable bloomfilter.
+    opt.options.set_memtable_prefix_bloom_ratio(0.02);
```

Comment:
We might also want to set `cache_index_and_filter_blocks_with_high_priority` and `pin_l0_filter_and_index_blocks_in_cache` as per the wiki here
https://github.com/EighteenZi/rocksdb_wiki/blob/master/Block-Cache.md

Reply:
It seems `cache_index_and_filter_blocks_with_high_priority` already defaults to `true` so only updated the other option.